github3/repos/branch.py

```diff
@@ -38,6 +38,26 @@ def _update_attributes(self, branch):
     def _repr(self):
         return '<Repository Branch [{0}]>'.format(self.name)
 
+    def latest_sha(self, differs_from=''):
+        """Check if SHA-1 is the same as remote branch
+
+        See: https://git.io/vaqIw
+
+        :param differs_from string: (optional), sha to compare against
+        :returns: strring of the SHA or None
+        """
+        # If-None-Match returns 200 instead of 304 value does not have quotes
+        headers = {
+            'Accept': 'application/vnd.github.chitauri-preview+sha',
+            'If-None-Match': '"{0}"'.format(differs_from)
+        }
+        base = self._api.split('/branches', 1)[0]
+        url = self._build_url('commits', self.name, base_url=base)
+        resp = self._get(url, headers=headers)
+        if resp:
```

Comment:
This could be more simply:

```
if resp and self._boolean(resp, 304, 200):
    return resp.content
return None
```

Also, I think `_get` will never return anything other than a requests Response so I don't think we need to check for its truthy-ness here.